core/jazz_azure-create-service/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "jazz-azure-cli",
+  "version": "1.0.4",
+  "description": "A jazz cli tool that allows creating and deleting api gateways. This is used internally by the Jazz framwork.",
+  "license": "MIT",
+  "preferGlobal": true,
+  "bin": {
+    "jazz-azure-cli": "bin/jazz-azure-cli"
+  },
+  "engines": {
+    "node": ">=8"
+  },
+  "keywords": [
+    "jazz",
+    "azure",
+    "cli",
+    "api gateway"
+  ],
+  "scripts": {},
+  "devDependencies": {},
+  "dependencies": {
+    "axios": "^0.18.0",
+    "azure-arm-apimanagement": "^4.4.0-preview",
+    "azure-arm-resource": "^7.3.0",
+    "azure-arm-storage": "^7.0.0",
+    "azure-arm-website": "^5.7.0",
+    "escape-json-node": "^2.0.0",
+    "form-data": "^2.3.3",
+    "minimist": "^1.2.0",
+    "ms-rest-azure": "^2.5.9",
+    "request": "^2.88.0",
```

Comment:
do you use both request and axios?  I think they are the both rest client.  All the current jazz code using request or http not axios.  just FYI